scripts/h5toms.py

```diff
@@ -60,10 +60,8 @@
                   help="More verbose progress information")
 parser.add_option("-w", "--stop-w", action="store_true", default=False,
                   help="Use W term to stop fringes for each baseline")
-parser.add_option("-x", "--HH", action="store_true", default=False,
-                  help="Produce a Stokes I MeasurementSet using only HH")
-parser.add_option("-y", "--VV", action="store_true", default=False,
-                  help="Produce a Stokes I MeasurementSet using only VV")
+parser.add_option("-p", "--pols-to-use", default=None,
+                  help="Select polarisation products to include in MS from HH,VV,HV,VH, default is all available from HH,VV")
```

Comment:
Yup, a comma separated list (I'll update the help text to be a bit more clear).

Indeed you can go HV,VH if you so desire- giving the user the freedom to do this was the reason for the change in the first place:)

The order shouldn't matter as its dealt with by the populate_polarisation_dict function. However I think that in all cases the order going into the MS will be HH, HV, VH, VV (which is what it was before) due to np.unique sorting the elements of its output array at line 125.

I tried making measurement sets in pathological cases like ones with only HV,VH as you mentioned above- and also ones with different ordering. CASA seemed happy with the files. BTW- XX is HH and YY is VV. This was already hard-coded in the populate_polarisation_dict function.